tensorflow_datasets/image/lfw.py

```diff
@@ -0,0 +1,105 @@
+from __future__ import absolute_import
+from __future__ import division
+from __future__ import print_function
+
+
+import os
+import tensorflow as tf
+import tensorflow_datasets.public_api as tfds
+
+_URL="http://vis-www.cs.umass.edu/lfw/lfw.tgz"
+LFW_IMAGE_SHAPE=(250,250,3)
+LFW_CITATION="""@TechReport{LFWTech,
```

Comment:
You can edit your line like others for better readability.
Use `LFW_CITATION="""\`